salt/_modules/etcd_discovery_helpers.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+import logging
+
+
+LOG = logging.getLogger(__name__)
+
+
+def __virtual__():
+    return "etcd_discovery_helpers"
+
+
+def get_etcd_cluster_size():
```

Comment:
I would rename this function to `get_cluster_size`, as we have to invoke it with  `'etcd_discovery_helpers.get_etcd_cluster_size` and we already know this is related to `etcd`...